pkg/storaged/stratis-details.jsx

```diff
@@ -804,13 +937,25 @@ const StratisStoppedPoolSidebar = ({ client, uuid }) => {
 };
 
 export const StratisStoppedPoolDetails = ({ client, uuid }) => {
+    const stopped_props = client.stratis_manager.StoppedPools[uuid];
+    const clevis_info = stopped_props.clevis_info;
+
+    const can_tang = clevis_info && clevis_info.v[0] && (!clevis_info.v[1].v[0] || clevis_info.v[1].v[1][0] == "tang");
+    const tang_url = can_tang && clevis_info.v[1].v[0] ? JSON.parse(clevis_info.v[1].v[1][1]).url : null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230522-113149-7b838e5e-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)